galaxy_importer/loaders/doc_string.py

```diff
@@ -79,8 +79,35 @@ def _get_plugins(self, plugin_dir):
                 fq_name_parts.append(os.path.basename(file_path)[:-3])
 
                 plugins.append(".".join(fq_name_parts))
+
         return plugins
 
+    def _run_ansible_doc_list(self, plugin_type):
+        """Use ansible-doc to get a list of plugins for the collection by type."""
+        collections_path = "/".join(self.path.split("/")[:-3])
+        cmd = [
+            "/usr/bin/env",
+            f"ANSIBLE_COLLECTIONS_PATHS={collections_path}",
```

Comment:
`ANSIBLE_COLLECTIONS_PATHS` is deprecated and will be removed in ansible-core 2.19. You only need `ANSIBLE_COLLECTIONS_PATHS` for backwards compatibity with Ansible 2.9.
```suggestion
            f"ANSIBLE_COLLECTIONS_PATH={collections_path}",
```